grass7/imagery/i.sentinel/i.sentinel.coverage/i.sentinel.coverage.py

```diff
@@ -281,11 +314,13 @@ def main():
     fpsize = get_size(fps_in_area_dis)
 
     percent = fpsize / areasize * 100.0
-    grass.message(_("%.2f percent of the area <%s> is covered") % (percent, area))
-
+    grass.message(_(
+        "%.2f percent of the area <%s> is covered") % (percent, area))
```

Comment:
I am not sure if it is ideal to break lines within a gettext macro... (also above and below)

Reply:
Is there a quick way to check whether it is recognized as a gettext macro? If the line gets too long it is not compliant with PEP8 any more.

Reply:
You could (probably should) use `format` instead `%`, inside brackets you should be able to split the line where you want

Reply:
Thanks, all string formatting in gettext macros were now replaced by `.format()`. If I understand correctly, the 
```
grass.message(_(
    "something"))

Reply:
should not be a problem, if you test the changes (compile and run command) and no problem it means no problem